src/implementation/Client/HTTPClient/HTTPClient.ts

```diff
@@ -213,6 +213,7 @@ export default class HTTPClient implements IClient {
           error_msg: `An unknown problem occurred and we got the status ${res.status} with response ${JSON.stringify(
             res,
           )}`,
+          error_detail: txt,
```

Comment:
```suggestion
          error_msg: txt,
```

In favor of this instead! `error_msg` by itself is not helpful today, since status is already reported and `res` is not useful. WDYT?

Reply:
Done!! good idea